wordfreq/tokens.py

```diff
@@ -88,51 +116,53 @@ def remove_arabic_marks(text):
 
 mecab_tokenize = None
 jieba_tokenize = None
```

Comment:
Now that `mecab_tokenize` and `jieba_tokenize` are set, not in the following function, but in functions above this one, can these two lines be moved to be above those two functions?  (It doesn't affect functionality, just readability.)